tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp

```diff
@@ -52,6 +52,8 @@ struct DuckDBPyRelation {
 
 	void Close();
 
+	unique_ptr<DuckDBPyRelation> GetAttribute(const string &name);
```

Comment:
I assume this just throws a `TypeError` if you do `relation[None]`?

Reply:
Yea it can't implicitly convert it to string, but might be worth a test 👍